server/mdm/android/service/pubsub.go

```diff
@@ -51,22 +51,28 @@ func (svc *Service) ProcessPubSubPush(ctx context.Context, token string, message
 	}
 
 	switch notificationType {
-	case android.PubSubEnrollment:
+	case string(android.PubSubEnrollment):
```

Comment:
Nit, but seems it would be a bit nicer to `switch NotificationType(notificationType)` and then the `case` don't need a type conversion?